packages/valory/skills/market_manager_abci/behaviours.py

```diff
@@ -154,6 +163,11 @@ def _update_bets(
             if self.synced_time >= bet.openingTimestamp - self.params.opening_margin:
                 bet.blacklist_forever()
 
+        # Extract the last queue number from the bets
+        # This is used to determine the next queue number
+        if self.bets:
+            self.current_queue_number = max(bet.queue_no for bet in self.bets)
```

Comment:
Isn't this the same as `self.current_queue_number -= 1`? Also, does the logic change if this gets removed completely since `current_queue_number` is never accessed again after this point?

Reply:
What confuses me is that this part of the code comes after using the value here:
https://github.com/valory-xyz/trader/blob/6e580904e713aebd0f3aece152c8bccd89d8663e/packages/valory/skills/market_manager_abci/behaviours.py#L156

and then next time it is used again it takes a new value here:
https://github.com/valory-xyz/trader/blob/6e580904e713aebd0f3aece152c8bccd89d8663e/packages/valory/skills/market_manager_abci/behaviours.py#L148